gutdata.gemspec

```diff
@@ -37,20 +37,18 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'yard-rspec', '~> 0.1'
   s.add_development_dependency 'ZenTest', '~> 4.10', '>= 4.11.0'
 
-  s.add_dependency 'aws-sdk-v1', '~> 1.45'
   s.add_dependency 'docile', '~> 1.1', '>= 1.1.5'
   s.add_dependency 'erubis', '~> 2.7', '>= 2.7.0'
   s.add_dependency 'gli', '~> 2.13', '>= 2.13.2'
+  s.add_dependency 'hashie', '>= 1.2.0', '< 4.0'
```

Comment:
haha why does this gem even exist? i'm not sure i've ever once thought, "gee, ruby hashes just aren't enough for me" 😆